holmes/plugins/toolsets/__init__.py

```
@@ -76,18 +78,29 @@ def load_python_toolsets(dal: Optional[SupabaseDal]) -> List[Toolset]:
         GitToolset(),
     ]
 
+    if not USE_LEGACY_KUBERNETES_LOGS:
+        toolsets.append(KubernetesLogsToolset())
+
     return toolsets
 
 
 def load_builtin_toolsets(dal: Optional[SupabaseDal] = None) -> List[Toolset]:
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)